src/components/graph/graph.renderer.jsx

```diff
@@ -59,19 +59,27 @@ function _buildLinks(nodes, links, linksMatrix, config, linkCallbacks, highlight
  * @returns {Object} returns the generated array of nodes components
  * @memberof Graph/helper
  */
-function _buildNodes(nodes, nodeCallbacks, config, highlightedNode, highlightedLink, transform) {
-    return Object.keys(nodes).map(nodeId => {
-        const props = buildNodeProps(
-            Object.assign({}, nodes[nodeId], { id: `${nodeId}` }),
-            config,
-            nodeCallbacks,
-            highlightedNode,
-            highlightedLink,
-            transform
-        );
+function _buildNodes(nodes, nodeCallbacks, config, highlightedNode, highlightedLink, transform, linksMatrix) {
+    return Object.keys(nodes)
+        .filter(
+            nodeId =>
+                !!Object.values(linksMatrix[nodeId]).reduce(
```

Comment:
Maybe move this to a function `countActiveNeighbours`?

Reply:
Sounds good!

Reply:
I've created a `getNodeCardinality` in `graph.helper.js`. 

Tell me what you think about the name. I felt that this way the name doesn't bind it directly to the concept of being `active`, but rather knowing how many nodes it has attached.

Reply:
But aren't we hiding the purpose of the function by detaching the name from `knowing how many nodes it has attached`?

Reply:
We discussed this and the conclusion was that the name does make sense as being `getNodeCardinality` :)